jenkins/image_building/build-image.sh

```diff
@@ -3,15 +3,21 @@
 set -eux
 
 current_dir="$(dirname "$(readlink -f "${0}")")"
+REPO_ROOT="$(realpath "${current_dir}/../..")"
 
 cleanup() {
   deactivate || true
+  cd "${REPO_ROOT}" || true
   sudo rm -f "${img_name}".{raw,qcow2}
+  sudo rm -rf "${img_name}.d"
```

Comment:
I do not understand this line ! Is it deleting a directory or an extension of the image

Reply:
There's an extra directory created by diskimage-builder and needs to be cleaned as well. Sorry I missed that in the PR description.